utils/resb/examples/genrb.rs

```diff
@@ -0,0 +1,36 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::{
+    fs::File,
+    io::{BufReader, BufWriter, Read, Write},
+};
+
+use resb::{binary, text};
+
+fn main() {
+    let input = File::open("examples/data/zoneinfo64.txt");
+    let mut reader = BufReader::new(input.unwrap());
+
+    let mut in_string = String::new();
+    match reader.read_to_string(&mut in_string) {
+        Ok(_) => (),
+        Err(err) => panic!("Unable to read file: {}", err),
+    };
```

Comment:
You can use https://docs.rs/main_error/latest/main_error/ to error out of main gracefully in all places where you handle errors manually now.

Reply:
I didn't find this as a dep or dev dep in the current tree, so I haven't added it for now.